File: Java/src/com/Wizard.java
package com;

public class Wizard{
    int power = 1, fairies = 10;

    public int getPower() { return power; }

    public void setPower(int power) { this.power = power; }

    public int getFairies() { return fairies; }

    public void setFairies(int fairies) { this.fairies = fairies; }

    public void move(boolean azar){
        System.out.println("Harry Potter found a fairy");
        if (azar) {
            this.setFairies(this.getFairies() + 3);
            if (this.getFairies() == 22)
                this.setPower(this.getPower() + 1);
            else
                System.out.println("The wizard get the fairy"+
				"\nThe wizard destroy the house");
        } else
            System.out.println("the don't get it jsjsjs");
    }
}

File: Python2/wizard.py
class Wizard:
    fairies = 10
    power = 1

    def __init__(self):
        super().__init__()

    def getFairies(self):
        return self.fairies

    def setFairies(self, _fairies):
        self.fairies = _fairies

    def setPower(self, _power):
        self.power = _power

    def getPower(self):
        return self.power

    def move(self, azar, fairy):
        print('Look, potter found a fairy')
        if azar:
            self.setFairies(self.getFairies() + fairy)
            if self.getFairies() >= 20:
                self.setPower(self.getPower() + 1)
            else:
                print('The wizard get a fairy\nAnd he destroy her house :c')
        else:
            print('HAHAHA so dumb he lose it')
